crates/pet-core/Cargo.toml

```diff
@@ -8,7 +8,6 @@ msvc_spectre_libs = { version = "0.1.1", features = ["error"] }
 
 [dependencies]
 pet-fs = { path = "../pet-fs" }
-pet-python-utils = { path = "../pet-python-utils" }
```

Comment:
Had to remove this incorrect depenedency, core should not have any other dependencies related to python interpreters and the like.
Hence the large number of files touched.